packages/web/components/cards/app-display-card.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FunctionComponent, memo } from "react";
+
+import { EventName } from "~/config";
+import { useAmplitudeAnalytics } from "~/hooks";
+
+import { Icon } from "../assets";
+import { IconLink } from "./icon-link";
+
+export const AppDisplayCard: FunctionComponent<{
+  title?: string;
+  subtitle?: string;
+  imageUrl?: string;
+  twitterUrl?: string;
+  githubUrl?: string;
+  externalUrl?: string;
+  mediumUrl?: string;
+  index: number;
+}> = memo(
+  ({
+    title,
+    subtitle,
+    imageUrl,
+    twitterUrl,
+    githubUrl,
+    externalUrl,
+    mediumUrl,
+    index,
+  }) => {
+    const { logEvent } = useAmplitudeAnalytics();
+    const handleAppClicked = () => {
+      logEvent([
+        EventName.AppStore.appClicked,
+        { appName: title, isFeatured: false, isBanner: false, position: index },
+      ]);
+    };
+    return (
+      <>
+        <style jsx>{`
+          .app-display-card:hover .card-image {
+            transform: scale(1.15);
+          }
+          .card-image {
+            background-image: url(${imageUrl});
+            transition: transform 0.3s ease-in-out;
+          }
```

Comment:
Are these still needed? I see you added some tailwind transitinos below. 

Above this `.app-display-card` could also be implemented with tailwind.

Reply:
I'm not sure tailwind alone can effect a transform on a child div when hovering over the parent. I had to add this specifically due to the product requirement of the img zooming in when you hover anywhere on the card. 

The extra transform is redundant, but as far as I can tell the transform on `.app-display-card` is needed

Reply:
I think the group utility classes are meant to fill this need. But it may be a plugin and may require some changes in the tailwind config. Given how uncommon this use case has been so far, let's leave this as is for now and we can re-enable group tags later as needed.

https://tailwindcss.com/docs/plugins#parent-and-sibling-states